SPECS/xapi.spec

```diff
@@ -1368,6 +1370,9 @@ Coverage files from unit tests
 %{?_cov_results_package}
 
 %changelog
+* Thu Mar 20 2025 Teddy Astie <teddy.astie@vates.tech> - 24.39.1-2
```

Comment:
you cannot change the version like that. It needs to be synchronized with the string in *Release:*. And also I'm not sure if it will be a 24.39.1-1.2 (need to check with platform I guess)